game/forms.py

```
@@ -52,4 +58,4 @@ def send_mail(
                 to_email,
                 html_email_template_name)
         except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)